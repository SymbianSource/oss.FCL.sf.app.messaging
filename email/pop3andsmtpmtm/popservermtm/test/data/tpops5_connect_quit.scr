+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 7 46040
+OK 7 messages
1 4901
2 5645
3 7829
4 7460
5 4885
6 7844
7 7476
.
+OK 7 messages
1 psion8.952427949:20:13434:10
2 psion8.952428159:20:19143:16
3 psion8.952362190:20:03972:5
4 psion8.952362192:20:03229:41
5 psion8.952362180:20:03922:1
6 psion8.952428087:11:10553:7
7 psion8.952428106:11:27774:203
.
-ERR Unknown Command "CAPA"
** Read cancelled
+OK 
+OK 
+OK 
+OK 
+OK 
+OK 
+OK 
+OK 
