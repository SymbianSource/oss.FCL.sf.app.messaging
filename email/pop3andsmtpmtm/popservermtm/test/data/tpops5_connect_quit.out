STAT
LIST
UIDL
CAPA
DELE 7
DELE 6
DELE 5
DELE 4
DELE 3
DELE 2
DELE 1
QUIT
